cl/opinion_page/templates/docket_tabs.html

```diff
@@ -71,33 +79,33 @@ <h1 class="bottom inline">
 
     <h2>{{ docket.court }}</h2>
 
-    {% if docket.pacer_url or docket_entries %}
+    {% if docket.pacer_docket_url or docket_entries %}
       <div class="v-offset-above-2">
-        {% if docket.pacer_url %}
-          <div class="left">
-            <p><a href="{{ docket.pacer_url }}"
-               class="btn btn-primary"
-               target="_blank"
-               rel="nofollow"><i class="fa fa-external-link"></i> View on PACER</a>&nbsp;</p>
-          </div>
-        {% endif %}
         {% if docket_entries %}
-          <div class="left">
-            <p><a href="/?type=r&amp;q=docket_id%3A{{ docket.pk }}"
-               class="btn btn-success"><i class="fa fa-search"></i>&nbsp;Search this Docket</a>&nbsp;</p>
-          </div>
+          <a href="/?type=r&amp;q=docket_id%3A{{ docket.pk }}"
+             class="btn btn-success"><i class="fa fa-search"></i>&nbsp;Search this Docket</a>
         {% endif %}
-        <div class="left">
+        <div class="btn-group"
+             id="react-root"
+             data-docket="{{ docket.id }}"
+             data-user-id="{% if user.is_authenticated %}{{ user.pk }}{% endif %}"
+             data-user-name="{% if user.is_authenticated %}{{ user.get_username }}{% endif %}"
+             data-authenticated="{% if user.is_authenticated %}{{ user.pk }}:{{ user.get_username }}{% endif %}"
+             data-edit-url="{% url "tag_list" request.user %}"
+             data-is-page-owner=""
```

Comment:
It's a bit weird we have this here, right?

Reply:
which part do you think is weird?

Reply:
this is how I elegantly pass the data to the React side